Rust/src/rcf.rs

```diff
@@ -63,7 +83,50 @@ pub trait RCF {
         score_unseen: fn(usize, usize) -> f64,
         damp: fn(usize, usize) -> f64,
         normalizer: fn(f64, usize) -> f64,
-    ) -> f64;
+    ) -> f64 {
+        self.score_visitor_traversal(point,&VisitorInfo::use_score(ignore_mass,score_seen,score_unseen,damp,normalizer))
+    }
+
+    fn score_visitor_traversal(&self, point :&[f32], visitor_info : &VisitorInfo) -> f64;
+
+    fn attribution(&self, point : &[f32]) -> DiVector {
+        self.attribution_visitor_traversal(point,&VisitorInfo::default())
+    }
+
+    fn generic_attribution(
```

Comment:
Do we need this function? Is it really more convenient than letting users create their own VisitorInfo struct with the functions they want?

Reply:
:) calling a function with different parameters (even if those parameters are functions) seems simpler than writing code -- specially in rust when the new code can start issues with the borrow checker/compiler